server.go

```diff
@@ -72,6 +73,8 @@ func (s *server) Serve() chan error {
 	mux.HandleFunc("/multihash/", s.find)
 	mux.HandleFunc("/providers", s.providers)
 	mux.HandleFunc("/health", s.health)
+	mux.Handle("/metrics", metrics.Start(nil))
```

Comment:
We should instantiate a separate server for these. As is:
* the will be publicly exposed via ingress.
* at ingress it will not be clear which pod they are coming from since it will be loadbalaced across currently 10 instances.